tests/src/Storage/ClickhouseTest.php

```diff
@@ -0,0 +1,165 @@
+<?php
+
+namespace Tests\Diag\Storage;
+
+use ClickhouseClient\Client\Client;
+use ClickhouseClient\Client\Config;
+use Diag\Exception\MissingRecord;
+use Diag\Exception\NotImplemented;
+use Diag\Record;
+use Diag\Storage\Clickhouse;
+use Diag\Storage\Sqlite;
+use PHPUnit\Framework\TestCase;
+
+class ClickhouseTest extends TestCase
+{
+    /** @var  Clickhouse */
+    protected $storage;
+    /** @var  Client */
+    protected $client;
+    public function setUp()
+    {
+        $config = new Config(
+            // basic connection information
+            ['host' => '127.0.0.1', 'port' => '5188', 'protocol' => 'http'],
```

Comment:
better to fetch test data from test parameters;

Reply:
I suggest doing this in a separate pull request. Otherwise we might get in too deep here..